src/Spout/Writer/Style/Style.php

```diff
@@ -302,6 +338,9 @@ public function mergeWith($baseStyle)
         if (!$this->hasSetWrapText && $baseStyle->shouldWrapText()) {
             $mergedStyle->setShouldWrapText();
         }
+        if (!$this->hasSetBackgroundColor && $baseStyle->shouldApplyBackgroundColor()) {
```

Comment:
Enabling setting a background without parameter seems weird. If you don't set any background, it should use the default background but if you do call the `setBackgroundColor()` method, I think your intent is to override the default behavior. Hence you should explicitly provide the new background color to use.

So what I would do is remove the concept of default bg color. And you can keep the if check as it is now